javascript/mobile.js

```diff
@@ -11,3 +11,13 @@ hamburger.addEventListener('click', (e) => {
     icon.classList.toggle('hidden');
   });
 });
+
+// to close menu when item is clicked
+const lists = document.querySelector('.menu-item');
+const spreadList = [...lists.children];
+spreadList.forEach((list) => {
+  list.addEventListener('click', () => {
+    nav.classList.toggle('open');
+    close.style.display = 'none';
+  });
+});
```

Comment:
- [ ] Popup does not appear correctly when ı click on it. It is not fit to mobile version. Kindly fix it.
- [ ] OPTIONAL. Under the page has a scroll bar in the mobile version. It should fit the screen.

![image](https://user-images.githubusercontent.com/38635158/189214411-85d4f3ad-7c7a-4156-a1db-4ecb070a5ca1.png)

Reply:
Hi, @ahmetbozaci I don't understand the first comment. Can you kindly explain more via zoom? 🙏🏾🙏🏾

This one >>  "Popup does not appear correctly when ı click on it. It is not fit to mobile version. Kindly fix it."

Reply:
@julzedz  Messaged in the slack.